### ISSUE TITLE
[TeamsChannel - Save As Dialogue Box]: Users is not able to access/select the list items of 'Switch view options' menu item in browse mode of NVDA.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13301]
https://user-images.githubusercontent.com/93735775/152244602-a688bb9f-47f5-4ef5-9354-d26ffac1e656.mp4

### Steps to reproduce:
1. Open the Teams app in Chrome and log in with valid credentials.
2. Navigate to the Teams tab in left pane and activate it.
3. Open any channel and navigate to files tab item and select it.
4. Enable NVDA in browse mode using 'Insert +Spacebar'.
5. Navigate to 'All Documents menu item' and activate it.
6. Try to navigate to list items of 'Switch view options' menu item.
7. Observe the issue.

### Actual behavior:
Users is not able to access/select the list items of 'Switch view options' menu item in browse mode of NVDA.
In Application mode (Normal mode) user is able to navigate over the listed items but after enabling browse mode, NVDA focus move to other background controls.

### Expected behavior:
Users should be able to access/select the list items of 'Switch view options' menu item using down arrow key in browse mode of NVDA and when user presses Enter, then the list item should also get selected.

### Note:
Issue is not reproducible with JAWS and NVDA.

### System configuration:
Windows 11
Chrome: Version 97.0.4692.99

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version 2021.3.1

#### Windows version:
OS: Windows 11 (Version 21H2 Build 22000.434)

#### Does the issue still occur after restarting your computer?
Yes

**Comment by seanbudd** [comment]
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers
